innersource-license.md

```diff
@@ -0,0 +1,83 @@
+## Title
+
+InnerSource License
+
+## Patlet
+
+Two companies that belong to the same group want to share software source code with each other but they are concerned about the implications in terms of legal liabilities or cross-company accounting. An **InnerSource License** provides a reusable legal framework for the inter-company agreement within the group, which opens up new collaboration options.
+
+## Problem
+
+When two companies within a group want to share code with each other, they need an agreement about the terms and often a legal contract. Creating such agreements on a per project basis takes effort and creates a barrier for sharing. i.e. a team/company might decide not to share their source code with another company in the group because it seems complicated.
+
+Barriers for sharing can lead to silos and duplication of effort in rebuilding similar solutions in multiple parts of the group.
+
+At the time of sharing the source code, it can not be reliably predicted what the value of sharing will be. Hence if the activity of sharing has any significant cost, the companies are unlikely to do it, as they are concerned about the return on investment.
```

Comment:
This is an excellent description of why this problem is a problem 👍

Reply:
Some sugestion for word smithing (feel free to commit, steal parts, or ignore)
```suggestion
At the time of sharing the source code, it can not be reliably predicted what the value of sharing will be. Hence if the activity of sharing creates significant effort (i.e. negotiating terms for the usage), the companies are less likely to do it: They cannot predict if the effort will pays off.
```

Reply:
Great feedback. I worked in something similar.